src/quo2/components/settings/settings_item/view.cljs

```diff
@@ -62,7 +62,8 @@
   (case tag
     :positive [status-tags/status-tag
                {:status          {:type :positive}
-                :label           (i18n/label :t/positive)
+                :label           (:label tag-props)
```

Comment:
status tags use a custom label not just positive/ negative 👍

Reply:
@J-Son89 there is context tags, or is this something different?

Reply:
This text should be dynamic - that's how they are using it across the screens.

Reply:
see this use case - https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?type=design&node-id=675-177571&mode=design&t=iEkepNLbrCfAzTX3-4

Reply:
Oh I see thanks!